cypress/e2e/seededFlows/articleFlows/playOrPauseAnimatedImages.spec.js

```diff
@@ -27,6 +27,7 @@ describe('play or pause animated images', () => {
 
   describe('no reduced motion preference', () => {
     beforeEach(() => {
+      cy.testSetup();
```

Comment:
This test need some cache clearing to take place, or it will fail.

This flaky spec has ruined my day.